loops_assignment.js

```diff
@@ -0,0 +1,75 @@
+1. Question
+// let num = 128
+// while (num >= 1) {
+//   console.log(num)
+//   num -= 1
+// }
+//
+//
+// for (let num = 124; num >= 1; num -= 1) {
+//   console.log(num)
+// }
+// ////////////////////////////////////////
+2. Question
+// let num = 0
+// while(num <= 100) {
+//   if(num % 2 === 0) {
+//     console.log(num + ' is even')
+//   } else {
+//     console.log(num + ' is odd')
+//   }
+//   num++
+// }
+// //////////////////////////////////////
+// for (let num = 0; num <= 100; num += 1) {
+//   if(num % 2 === 0) {
+//     console.log(num + ' is even')
+//   } else {
+//     console.log(num + ' is odd')
+//   }
+// }
+/////////////////////////////////////////
+3. Question
```

Comment:
So, for this one, we were actually asking you to multiply the number by 9, not to log it if it's divisible by 9.